### ISSUE TITLE
[Bug]: 1.20.1 Alpha, don't work for server side

### ISSUE THREAD

**Issue by DonorKrovi** [Alex-the-666/Ice_and_Fire#4969]
### What happened?

Start server = crash

### Minecraft Version

1.20

### Ice and Fire version

iceandfire-2.1.13-1.20.1+build.2.1.13-alpha-1

### Citadel version

2-5-4

### Relevant log output

```shell
https://gist.github.com/DonorKrovi/32ee02e48f8061355e83af717df3d4af
```

### Other Information

- [ ] I am using other mods besides Ice and Fire
- [X] I have attached a log file from the session that I have experienced this issue

**Comment by DonorKrovi** [comment]
Hmmm, was JEI an addiction to this mod before?

**Comment by TheBv** [comment]
Should be an optional dependency. Might just be that we are on an older version of JEI right now. (version 15.2.0.25) 
I'll make sure this gets fixed next alpha release!

**Comment by DonorKrovi** [comment]
> Should be an optional dependency. Might just be that we are on an older version of JEI right now. (version 15.2.0.25) I'll make sure this gets fixed next alpha release!

Many thanks

**Comment by TheBv** [comment]
Fixed with 97d6f5bd922721d5d0e2038a4a34d8bb8cdb8dd1
